server/lib/authorize.js

```diff
@@ -0,0 +1,40 @@
+const axios = require('axios');
```

Comment:
Not the biggest fan of having to files with the same name tbh. Sure, they're in different directories - but it can be a bit confusing when there's a route called the same thing. Maybe we could rename this one to `auth.js` or something more generic?

Reply:
We should be able to remove the other file completely anyway, let's do this in a separate PR that completely removes the original `/auth` endpoint